### ISSUE TITLE
Math Error?

### ISSUE THREAD

**Issue by bbakermmc** [mistic100/jQuery-QueryBuilder#788]
![image](https://user-images.githubusercontent.com/30090055/57871341-18d81880-77d7-11e9-9efc-f837f191b789.png)

```sql
ContactEventActivity.ActivityTypeId = 49 AND ( ContactEventResponse.WarmTransferStatusId = 10734 OR ContactEventResponse.WarmTransferStatusId IS NULL ) AND CompanyUniverse.UniverseId IN(-20, -21, -22) AND ( ContactEventResponse.FollowUpActionId != 10737 OR ( ContactEventResponse.SizeOfOpportunity IS NULL OR ContactEventResponse.SizeOfOpportunity <= 5000 ) OR ContactEventResponse.PurchaseTimeFrameImmediateId != 10735 ) 
```

**Comment by bbakermmc** [comment]
The sql was created with the tool, but when trying to reload it, getting math error.

**Comment by bbakermmc** [comment]
Seems like it doesn like negative numbers in the in clause for CompanyUniverse.UniverseId IN(-20, -21, -22)

**Comment by mistic100** [comment]
Fixed with SQL Parser 1.2.3
